discord/abc.py

```diff
@@ -552,12 +552,12 @@ def created_at(self) -> datetime:
         """:class:`datetime.datetime`: Returns the channel's creation time in UTC."""
         return utils.snowflake_time(self.id)
 
-    def overwrites_for(self, obj: Union[Role, User, Object]) -> PermissionOverwrite:
+    def overwrites_for(self, obj: Union[Role, User, Member, Object]) -> PermissionOverwrite:
         """Returns the channel-specific overwrites for a member or a role.
 
         Parameters
         -----------
-        obj: Union[:class:`~discord.Role`, :class:`~discord.abc.User`, :class:`~discord.Object`]
+        obj: Union[:class:`~discord.Role`, :class:`~discord.abc.User`, :class:`~discord.Discord`, :class:`~discord.Object`]
```

Comment:
I don't think `discord.Discord` is correct...?

Also `User` here infers `abc.User` which is the abstract base class that both `discord.User` and `discord.Member` adhere to.

Reply:
fix typo `discord.Discord`

as I see member does not inherits `abc.User`

```
class Member(discord.abc.Messageable, _UserTag):
```

Reply:
It does not need direct inheritance as it is a Snowflake | Protocol:-

<https://github.com/Rapptz/discord.py/blob/799e3c5bf09cedcd9b7e2b83df733910fa11532a/discord/abc.py#L199-L207>